build/Dockerfile

```diff
@@ -6,20 +6,22 @@ COPY . .
 
 RUN ATTACHER_TAG=libbpf make build
 
-FROM registry.access.redhat.com/ubi9-minimal:9.2
-RUN microdnf -y update
+FROM registry.access.redhat.com/ubi9
```

Comment:
change to `registry.access.redhat.com/ubi9:9.2` ?

Reply:
should we keep to use latest tag instead of any specific version for example 9.2 ?
as the image will build daily, use a latest tag for nightly build is better?

Reply:
btw, honestly, .... I hope some one can make it easier....
```
registry.access.redhat.com/ubi9
registry.access.redhat.com/ubi:9
registry.access.redhat.com/ubi9-minimal:9.2
registry.access.redhat.com/ubi9:9.2
```
what's the naming rule and difference?

Reply:
i think it is good practice not to use `latest` tag. as it is hard to rollback in case of issues.

Reply:
k, let's try.